countries/src/main/java/com/lambdaschool/countries/CheckCountry.java

```diff
@@ -0,0 +1,6 @@
+package com.lambdaschool.countries;
+
+public interface CheckCountry
```

Comment:
Awesome work creating the interface that allows you to use your lambda expression